.github/workflows/nighthawk_checks.yml

```diff
@@ -0,0 +1,20 @@
+name: Nighthawk Envoy CI/CD
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest  # Equivalent to your 'envoy-x64-large' pool
```

Comment:
this will give you the default github-hosted vm - you will need to use the label i posted `envoy-x64-medium` for larger machine

also - as a general point - best not to use `-latest` as its better to pin to a version and change it explicitly